README.md

```diff
@@ -197,6 +198,21 @@ Example:
 loginsrv -backend 'provider=htpasswd,file=users
 ```
 
+### Httpupstreem
+Authentication against an upstream http server by performing a simple simple authenticated request
```

Comment:
lol that's what the second 'simple' should have been "simple basic authentication request" - I probably should have gone to bed before submitting this.